sdk/src/Core/Amazon.Runtime/Internal/Auth/CloudFrontSigner.cs

```diff
@@ -40,6 +40,7 @@ public override void Sign(IRequest request, IClientConfig clientConfig, RequestM
             }
 
             string dateTime = AWSSDKUtils.GetFormattedTimestampRFC822(0);
```

Comment:
GetFormattedTimestampRFC822 calls AWSSDKUtils.CorrectedUtcNow internally, so instead of trying to parse the datetime just to get the time I just call it here.

Reply:
Why not do `CorrectClockSkew.GetClockCorrectionForEndpoint(request.Endpoint.ToString())` like the AWS4Signer does?

Reply:
Good catch, it actually seems like the `CloudFrontSigner` is not accounting for clockskew. `AWSSDKUtils.GetFormattedTimestampRFC822(0)` doesn't check `CorrectClockSkew`'s dictionary at all. I could change this to something like `CorrectClockSkew.GetCorrectedUtcNowForEndpoint(request.Endpoint.ToString()).ToString(AWSSDKUtils.RFC822DateFormat, CultureInfo.InvariantCulture)` but that would be a behavior change (even though it is the correct one).